core/node/da_dispatcher/src/metrics.rs

```diff
@@ -0,0 +1,26 @@
+use std::time::Duration;
+
+use vise::{Buckets, Gauge, Histogram, Metrics};
+
+#[derive(Debug, Metrics)]
+#[metrics(prefix = "server_da_dispatcher")]
+pub(super) struct DataAvailabilityDispatcherMetrics {
+    /// Latency of the dispatch of the blob.
+    #[metrics(buckets = Buckets::LATENCIES)]
```

Comment:
Please use `unit = Unit::Seconds` for metrics with unit of measurement defined. It's not defined for some older metrics for backward compatibility reasons (unit influences metric name).

Reply:
thanks, added unit